requirements.txt

```diff
@@ -1,7 +1,8 @@
 Django==4.2.4
-find-similar==1.0.1
+find-similar
 djangorestframework==3.14.0
 markdown2==2.4.10
 django-filter==23.2
 django-cors-headers==4.2.0
-drf-yasg==1.21.7
\ No newline at end of file
+drf-yasg==1.21.7
+nltk
```

Comment:
`nltk` installs with `find-similar`. Don't understand why should we duplicate it here?

Reply:
Stop_word means the words you want to skip like 
'is', "a", "the" etc

Lets take an example:

Comparable_txt = "Ali"
Text = ["Ali Ahmad", "is Ali"]
remove_stopwords = true

Here in above example, there is word "is" in text. We set remove_stopwords=true, this means the words that is useless will be removed. Hence the answer should be

Ali
1.0
Ali Ahmad
0.7

Problem:
When i run this example on find_simple with remove_stopwords it returned

Ali Ahmad
0.7
is Ali
0.7

See, useless word with no meaning which have to be removed still there. So i add the NLTK library to get prebuilt nltk.stop_words to handle this scenerio, and it runs successfully.

Reply:
As you said nltk is being used with find-simple, so i just remove this duplicate